Composer/packages/client/src/components/InstallDepModal.tsx

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable react/no-danger */
```

Comment:
what happened to NodeModal, if it's removed this PR doesn't have that change

Reply:
Removed that file.